components/collector/src/source_collectors/__init__.py

```diff
@@ -51,12 +51,12 @@
 from .github.merge_requests import GitHubMergeRequests
```

Comment:
Hmmmmm, but actually GitHub also has a view to show inactive branches!
However, they call them "stale": https://github.com/ICTU/quality-time/branches/stale

Reply:
Nice. We can use that when we (or @termeczky, wink) add GitHub as possible source for the inactive branches metric.